src/tools/Package-Core.py

```diff
@@ -187,10 +222,15 @@ def main(argv):
             merged_file_destination = os.path.join(working_directory, 'out', merged_file_detail[0])
             generate_compiled_script(source_code_path, merged_file_destination, merged_file_detail[0], merged_file_detail[1])
 
+        # add all dependencies under core/src/external_dependencies to destination directory
+        external_dependencies_destination = os.path.join(merge_file_directory, 'external_dependencies')
+        external_dependencies_source_code_path = os.path.join(source_code_path, 'external_dependencies')
+        add_external_dependencies(external_dependencies_destination, external_dependencies_source_code_path)
+
     except Exception as error:
-        print('Exception during merge python modules: ' + repr(error))
+        print('Exception during packaging all python modules in core: ' + repr(error))
         raise
 
 
 if __name__ == "__main__":
-    main(sys.argv)
+    main(sys.argv)
```

Comment:
What changed here? Ensure file ends with a new line. The IDE should have complained about this.